Adjust phpMyAdmin vendor configuration to match Debian needs:
 - setup generates configuration in /var
 - documentation is in /usr/share/doc
 - config file consists of several included files, so we skip mtime check
--- a/libraries/vendor_config.php
+++ b/libraries/vendor_config.php
@@ -15,30 +15,30 @@
  * Path to changelog file, can be gzip compressed. Useful when you want to
  * have documentation somewhere else, eg. /usr/share/doc.
  */
-define('CHANGELOG_FILE', './ChangeLog');
+define('CHANGELOG_FILE', '/usr/share/doc/phpmyadmin/changelog.gz');
 
 /**
  * Path to license file. Useful when you want to have documentation somewhere
  * else, eg. /usr/share/doc.
  */
-define('LICENSE_FILE', './LICENSE');
+define('LICENSE_FILE', '/usr/share/doc/phpmyadmin/copyright');
 
 /**
  * Path to config file generated using setup script.
  */
-define('SETUP_CONFIG_FILE', './config/config.inc.php');
+define('SETUP_CONFIG_FILE', '/var/lib/phpmyadmin/config.inc.php');
 
 /**
  * Whether setup requires writable directory where config
  * file will be generated.
  */
-define('SETUP_DIR_WRITABLE', true);
+define('SETUP_DIR_WRITABLE', false);
 
 /**
  * Whether to skip mtime check on config file and load it on every request.
  * This has slight performance drawback, but it is the only way to handle
  * inclusions of other php scripts from config.
  */
-define('SKIP_MTIME_CONFIG_CHECK', false);
+define('SKIP_MTIME_CONFIG_CHECK', true);
 
 ?>
--- a/Documentation.html
+++ b/Documentation.html
@@ -245,20 +245,10 @@
                 <a href="setup/">Setup Script</a>. First you must
                 manually create a folder <tt>config</tt> in the phpMyAdmin
                 directory. This is a security measure. On a Linux/Unix system you
-                can use the following commands:
+                can use the following command:
 <pre>
-cd phpMyAdmin
-mkdir config                        # create directory for saving
-chmod o+rw config                   # give it world writable permissions
-</pre>
-                And to edit an existing configuration, copy it over first:
-<pre>
-cp config.inc.php config/           # copy current configuration for editing
-chmod o+w config/config.inc.php     # give it world writable permissions
+/usr/sbin/pma-configure
 </pre>
-                On other platforms, simply create the folder and ensure that your
-                web server has read and write access to it. <a href="#faq1_26">FAQ
-                1.26</a> can help with this.<br /><br />
 
                 Next, open <tt><a href="setup/">setup/</a>
                 </tt>in your browser. Note that <strong>changes are not saved to
@@ -271,13 +261,10 @@
                 or use the <tt>Download</tt> link to save the config file locally
                 and upload (via FTP or some similar means) to the proper location.<br /><br />
 
-                Once the file has been saved, it must be moved out of the <tt>
-                config/</tt> directory and the permissions must be reset, again
+                Once the file has been saved, the permissions must be reset, again
                 as a security measure:
 <pre>
-mv config/config.inc.php .         # move file to current directory
-chmod o-rw config.inc.php          # remove world read and write permissions
-rm -rf config                      # remove not needed directory
+/usr/sbin/pma-secure
 </pre>
                 Now the file is ready to be used. You can choose to review or edit
                 the file with your favorite editor, if you prefer to set some
